modules/digital-twins/digital-twins-instance/endpoint--event-hub/main.bicep

```diff
@@ -92,3 +87,15 @@ output resourceGroupName string = resourceGroup().name
 
 @description('The name of the Endpoint.')
 output name string = endpoint.name
+
```

Comment:
As system assigned appears supported, let's add the corresponding principal ID output

Reply:
Same like for the other endpoint: It does not work. Linter displays a warning and also gives me an error during deployment if I try this despite the warning...

Reply:
This will be interesting during the migration as I think we have a spec & test for that. Ah well. Let's bridge that gap once we get there